config/configuration.toml.in

```diff
@@ -1,6 +1,6 @@
 # XXX: Warning: this file is auto-generated from file "@CONFIG_IN@".
 
-[hypervisor.qemu-lite]
+[hypervisor.q35]
```

Comment:
@sboeuf are you trying to build our latest q35? If so, I thought you'd be cloning from: https://github.com/clearcontainers/qemu/tree/qemu-lite-v2.9.0
(note the specific branch).
Maybe I mis-understand?